torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -1,7 +1,6 @@
 import torch
 from typing import Tuple, Optional, Union
-from collections import defaultdict
-import functools
+import torchao.ops
```

Comment:
I suspect this line broke our binary builds

Reply:
yeah is this caught by normal PR CI btw?